internal/service/networkpeering/data_source_network_peering.go

```diff
@@ -124,101 +125,49 @@ func dataSourceMongoDBAtlasNetworkPeeringRead(ctx context.Context, d *schema.Res
 		return diag.FromErr(fmt.Errorf(errorPeersRead, peerID, err))
 	}
 
-	accepterRegionName, err := ensureAccepterRegionName(ctx, peer, conn, projectID)
-	if err != nil {
-		return diag.FromErr(err)
-	}
-	if err := d.Set("accepter_region_name", accepterRegionName); err != nil {
-		return diag.FromErr(fmt.Errorf("error setting `accepter_region_name` for Network Peering Connection (%s): %s", peerID, err))
-	}
-	if err := d.Set("aws_account_id", peer.AWSAccountID); err != nil {
```

Comment:
don't we need all these Set?

Reply:
refactored these into a `setCommonFields` function. After comparing the resource and data_source